### PR TITLE
python3Packages.webexteamssdk: init at 1.6

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python module for Webex Teams APIs

https://github.com/CiscoDevNet/webexteamssdk

This is a Home Assistant dependency.

There are no tests available (thanks @mweinelt for checking).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/home-assistant/component-packages.nix [hunk 1]
```diff
@@ -115,7 +115,7 @@
     "circuit" = ps: with ps; [ ]; # missing inputs: circuit-webhook
     "cisco_ios" = ps: with ps; [ pexpect ];
     "cisco_mobility_express" = ps: with ps; [ ciscomobilityexpress ];
-    "cisco_webex_teams" = ps: with ps; [ ]; # missing inputs: webexteamssdk
+    "cisco_webex_teams" = ps: with ps; [ webexteamssdk ];
```
Comment (SuperSandro2000) [inline pkgs/servers/home-assistant/component-packages.nix:118]: Are there home-assistant tests we can enable?

**Comment by SuperSandro2000** [comment]
@mweinelt You're help with home-assistant tests is needed here :)

**Comment by mweinelt** [comment]
> @mweinelt You're help with home-assistant tests is needed here :)

Broken because of #110863.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 118809` at bd787ea5 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/52d4fdb0c27375e7321d89674de27357">python38Packages.webexteamssdk</a></li>
    <li><a href="https://gist.github.com/2b7fa8863f3702d8efa10d877334b342">python39Packages.webexteamssdk</a></li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement.

File: pkgs/development/python-modules/webexteamssdk/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+buildPythonPackage rec {
+  pname = "webexteamssdk";
+  version = "1.6";
+
+  src = fetchFromGitHub {
+    owner = "CiscoDevNet";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1m39kfmjh5bdymg2bn4vqf6jiknyniz669g4g5dh7v1sv93jxda7";
```
Comment (mweinelt) [inline pkgs/development/python-modules/webexteamssdk/default.nix:18]:
```
hash mismatch in fixed-output derivation '/nix/store/wnj87411rf892hfm71ms9adzi7aiw4r9-source':
  wanted: sha256:1m39kfmjh5bdymg2bn4vqf6jiknyniz669g4g5dh7v1sv93jxda7
  got:    sha256:0bw28ag1iqyqlxz83m4qb3r94kipv5mpf3bsvc8zv5vh4dv52bp2
```

**Comment by mweinelt** [comment]
> Looks like there is a bug in the detection of the [tests](https://github.com/home-assistant/core/tree/dev/homeassistant/components/cisco_webex_teams?rgh-link-date=2021-04-08T12%3A40%3A47Z) (#118703).

There is no `tests/components/cisco_webex_teams`, so the error is correct.

File: pkgs/servers/home-assistant/component-packages.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/servers/home-assistant/component-packages.nix:118]: No.

**Comment by fabaff** [comment]
> There is no tests/components/cisco_webex_teams, so the error is correct.

Note to myself: Check the correct location for the tests.

**Pull request merged**